../../../../../../../Library/Caches/com.apple.python/Users/naveenus/Projects/Upward/Backend/env/lib/python3.9/site-packages/supafunc/__init__.cpython-39.pyc,,
../../../../../../../Library/Caches/com.apple.python/Users/naveenus/Projects/Upward/Backend/env/lib/python3.9/site-packages/supafunc/_async/__init__.cpython-39.pyc,,
../../../../../../../Library/Caches/com.apple.python/Users/naveenus/Projects/Upward/Backend/env/lib/python3.9/site-packages/supafunc/_async/functions_client.cpython-39.pyc,,
../../../../../../../Library/Caches/com.apple.python/Users/naveenus/Projects/Upward/Backend/env/lib/python3.9/site-packages/supafunc/_sync/__init__.cpython-39.pyc,,
../../../../../../../Library/Caches/com.apple.python/Users/naveenus/Projects/Upward/Backend/env/lib/python3.9/site-packages/supafunc/_sync/functions_client.cpython-39.pyc,,
../../../../../../../Library/Caches/com.apple.python/Users/naveenus/Projects/Upward/Backend/env/lib/python3.9/site-packages/supafunc/errors.cpython-39.pyc,,
../../../../../../../Library/Caches/com.apple.python/Users/naveenus/Projects/Upward/Backend/env/lib/python3.9/site-packages/supafunc/utils.cpython-39.pyc,,
../../../../../../../Library/Caches/com.apple.python/Users/naveenus/Projects/Upward/Backend/env/lib/python3.9/site-packages/supafunc/version.cpython-39.pyc,,
supafunc-0.10.1.dist-info/INSTALLER,sha256=zuuue4knoyJ-UwPPXg8fezS7VCrXJQrAP7zeNuwvFQg,4
supafunc-0.10.1.dist-info/LICENSE,sha256=M03Wgg4urqsgZOfFkAG4EFZnKKKKQafB2_abvuF9CTY,1065
supafunc-0.10.1.dist-info/METADATA,sha256=pZRoF1cd7FT-GvcVTp2_lEhwEJIYtbe5j3wR-_t0c6s,1193
supafunc-0.10.1.dist-info/RECORD,,
supafunc-0.10.1.dist-info/WHEEL,sha256=Nq82e9rUAnEjt98J6MlVmMCZb-t9cYE2Ir1kpBmnWfs,88
supafunc/__init__.py,sha256=3FPolCnpQU4QfAQ7QKf-l-E387-kbgtIZq10tUJ2OR4,939
supafunc/_async/__init__.py,sha256=U4S_2y3zgLZVfMenHRaJFBW8yqh2mUBuI291LGQVOJ8,35
supafunc/_async/functions_client.py,sha256=Ei9CfSs3xb8D8fXn8cmHNeHhb2hSBJnEMW6QZ1TPD9I,5326
supafunc/_sync/__init__.py,sha256=U4S_2y3zgLZVfMenHRaJFBW8yqh2mUBuI291LGQVOJ8,35
supafunc/_sync/functions_client.py,sha256=7KWAo4PtJ9vXyq8i1o1mP6oa4cKEILqBkRQ0YeyaNLQ,5286
supafunc/errors.py,sha256=D65KK1Zkiuxg5Z-SnX1qVXU8hz8HN0ma__uvTtg7t7M,1101
supafunc/utils.py,sha256=EiZhFMFXeWzQHr1Cp7VqQcx-hp4HjBwZI6tTTW5gDvc,1603
supafunc/version.py,sha256=VIzlo3QVAoLTwSeBiCjegVA9WDOaEDh74TY2QAO5fqQ,53
